spec/system/searching_spec.rb

```diff
@@ -68,9 +68,9 @@
 
   # We don't have the on-site icon at the moment. It appears as available. We might add it back in the future.
   context 'with items which are from aeon locations' do
-    it 'renders an accessible warning icon for requesting an item in a reading room' do
+    it 'does not render a warning icon for requesting an item in a reading room' do
```

Comment:
Do we still need this test?

Reply:
I can remove them. I kept them to make sure the icon is not there anymore. But it can be deleted now